app/screens/Onboarding/SignUp/SignUpSuccess/SignUpSuccess.styles.ts

```diff
@@ -0,0 +1,26 @@
+import { ImageStyle, StyleSheet, ViewStyle } from 'react-native';
+
+interface ISignUpSuccessScreenStyles {
+  container: ViewStyle;
+  mt113: ImageStyle;
+  title: ViewStyle;
+  subtitle: ViewStyle;
+}
+
+export const styles = StyleSheet.create<ISignUpSuccessScreenStyles>({
+  container: {
+    flex: 1,
+    marginHorizontal: 24,
+  },
+  mt113: {
+    marginTop: 113,
+  },
```

Comment:
rename it properly
the style name shouldn't indicate how much margin is

Reply:
Not valid
After discussion, we decided to keep